### PR TITLE
Fix tests/seige/urlgen for non-macs

### PR DESCRIPTION
**Pull request by pcn**
Should address https://github.com/influxdb/influxdb/issues/1437

### CODE REVIEW

**Comment by otoolep** [comment]
I tested the changes on Ubuntu 14.04, they look good. However, leaving to @benbjohnson to make final merge.

File: tests/siege/urlgen [hunk 1]
```diff
@@ -50,7 +64,7 @@ for i in `seq 1 $NUMPOINTS`;
 do
 	# Move forward the current time.
 	let TIME=TIME+INTERVAL
```
Comment (otoolep) [inline tests/siege/urlgen]: Needs to be  `TIME=$(expr $TIMESTAMP + 1)` in bash.

**Comment by pcn** [comment]
Fixed

**Comment by benbjohnson** [comment]
@otoolep lgtm

**Comment by otoolep** [comment]
Thanks @pcn

**Comment by pauldix** [comment]
Awesome, thanks @pcn. Can you sign our CLA? http://influxdb.com/community/cla.html

**Comment by pcn** [comment]
Done.

**Pull request merged**